src/odemis/gui/cont/tabs.py

```diff
@@ -2939,33 +2939,18 @@ def _milling_angle_changed(self, pos):
         Updates the milling control with the changed angle position.
         :param pos: (float) value of rx
        """
-        rx_value = math.degrees(pos)
-        self.panel.ctrl_rx.Value = readable_str(rx_value, unit="°", sig=3)
+        current_angle = math.degrees(pos)
```

Comment:
Let's add a comment about the us of `HasFocus()`. Something like: "When the user is typing a new value in, it's not handy that the text changes, so don't update in such case".